### DISCUSSION TITLE
Axmol release strategy

### DISCUSSION THREAD

**Discussion by halx99** [axmolengine/axmol#2153] [Announcements]
Since axmol 2.1.5, we have new release strategy:

1. Increase minor version if the release contains features or improment
2. Increase patch only if the release only contains bugfixs
3. All 3rdparty or sdks tools no longer update in individual commit or PR, once we ready release a new version, we will make a branch `release-x.y.z` to create a PR to update 3rdparty or sdks to latest stable release, merge first, wait some days, will create second PR for final release

Note:
- Next release will be 2.2.0 instead 2.1.6
- axmol 2.x now is LTS

**Comment by smilediver** [comment]
Maybe it would be better to update 3rd party stuff in dev branch right after the release, unless we need some urgent fixes? This way it would get more testing. Updating right before release seems more risky.
